### PR TITLE
Security updates

### PR DESCRIPTION
**Pull request by craigkerstiens**

### CODE REVIEW

**Comment by begriffs** [comment]
Fixes #507

File: cloud/security.rst [hunk 1]
```diff
@@ -126,3 +126,31 @@ All data within Citus Cloud is encrypted at rest, including data on the instance
   <script type="text/javascript">
   analytics.track('Doc', {page: 'Security', section: 'cloud'});
   </script>
+
+Network perimeter controls
+==========================
+
+All Citus clusters run in their own isolated Amazon Virtual Private Cloud (VPC). In addition to all security guarantees which are on by default you can also secure your citus cluster by limiting network access. There are two methods for limiting network access:
```
Comment (lfittl) [inline cloud/security.rst]: Upcase `citus clusters` to be `Citus clusters`

File: cloud/security.rst [hunk 2]
```diff
@@ -126,3 +126,31 @@ All data within Citus Cloud is encrypted at rest, including data on the instance
...
+2. IP whitelisting
+
+VPC Peering
+-----------
+
+A VPC peering connection is a networking connection between two VPCs that
+enables you to route traffic between them using private IPv4 addresses or IPv6
+addresses. Instances in either VPC can communicate with each other as if they
+are within the same network. In order to setup a VPC peering connecting between
+your existing Citus cloud formation and your existing Amazon VPC open a support
```
Comment (lfittl) [inline cloud/security.rst]: Probably change `Citus cloud` to `Citus Cloud` since we use it that way elsewhere I think

File: cloud/security.rst [hunk 3]
```diff
@@ -126,3 +126,31 @@ All data within Citus Cloud is encrypted at rest, including data on the instance
...
+ticket and then we will initiate the peering request.
+
+IP Whitelisting
+---------------
+
+IP whitelisting allows you to restrict your Citus cloud cluster so that only the
+listed IPs are able to connect to your cluster. IP whitelisting is available for
+a single IP or a set of IP ranges. In order to enable IP whitelisting on your
+Citus cloud formation open a support ticket with the desired IP ranges and we'll
+get it setup for you.
```
Comment (lfittl) [inline cloud/security.rst]: Same `Citus cloud` to `Citus Cloud` change here

File: cloud/security.rst (hunk 1 above)
Comment (begriffs) [reply cloud/security.rst]: Yeah I'll be giving this pull request an editing pass. Craig opened the first draft for our convenience.

**Comment by craigkerstiens** [comment]
I'm good for us to ship this

**Pull request merged**